packages/@sanity/form-builder/src/inputs/files/common/ActionsMenu.tsx

```diff
@@ -45,7 +45,7 @@ export function ActionsMenu(props: Props) {
       {browse}
 
       <MenuDivider />
-      <MenuItem as="a" icon={DownloadIcon} text="Download original file" href={downloadUrl} />
+      <MenuItem as="a" icon={DownloadIcon} text="Download" href={downloadUrl} />
```

Comment:
Same as the above comment - this doesn't seem like it belongs to the PR, just checking if it was intentionally included here.

Reply:
Same as before, I think it was because the text was considered too long? @jorngeorg

Reply:
I edited it because it's factually wrong. As long as we're not using `dl=raw` you're not downloading the original but a an altered copy of the image.